repositories/repositories.bzl

```diff
@@ -233,3 +233,11 @@ def ros2_repositories():
         strip_prefix = "unique_identifier_msgs-2.1.3",
         urls = ["https://github.com/ros2/unique_identifier_msgs/archive/2.1.3.tar.gz"],
     )
+
+    maybe(
+        new_git_repository,
+        name = "ros2_gps_msgs",
+        build_file = "@com_github_mvukov_rules_ros2//repositories:gpsfix_msgs.BUILD.bazel",
+        remote = "https://github.com/swri-robotics/gps_umd.git",
+        branch = "ros2_devel",
+    )
```

Comment:
It's a better practice to stick to a particular commit and just put the branch name in a comment. Here I put another proposal with some inline comments that you can leave out later.
```
# ros2-devel branch
maybe(
    http_archive,
    # Should be the same as the original repo name + the prefix. ros2_ prefix is to
    # e.g. enable using another commit for ROS1 rules_ros.
    # rules_ros and rules_ros2 should be able to work in a another/joint Bazel workspace.
    name = "ros2_gps_umd",
    # Please rename the build file to the repo name for easier referencing.
    build_file = "@com_github_mvukov_rules_ros2//repositories:gps_umd.BUILD.bazel",  
    sha256 = "<calculate sha sum of the archive using sha256sum>",
    strip_prefix = "gps_umd-fc782811804fafb12ee479a48a2aa2e9ee942e2d",
    urls = ["https://github.com/swri-robotics/gps_umd/archive/fc782811804fafb12ee479a48a2aa2e9ee942e2d.tar.gz"],
)
```